examples/daisyworld.jl

```diff
@@ -48,249 +60,269 @@ mutable struct Daisy <: AbstractAgent
-    collect_model_data!(model_df, model, mdata, t)
-    collect_agent_data!(agent_df, model, adata, t)
-    heatmap(
-        1:model.space.dimensions[1],
-        1:model.space.dimensions[2],
-        transpose(reshape(model.temperature, model.space.dimensions));
-        clims = (-50, 110),
-        colorbar_title = "Temperature",
-    )
-    scatter!(
-        [a.pos for a in allagents(model)];
-        marker = (:circle, 5),
-        markercolor = [a.breed for a in allagents(model)],
-        label = :none,
-        showaxis = false,
-    )
-end
-gif(anim, "daisyworld.gif", fps = 10)
-
-# Very interesting! But why is this all happening? Luckily we have collected some useful
-# data, so now if we plot our different properties over the same time period, we can see
-# how each of the values effect Daisyworld as a whole.
-
-p1 = plot(model_df[!, :solar_luminosity], legend = false, ylabel = "Solar Luminosity")
-p2 = plot(model_df[!, :global_temperature], legend = false, ylabel = "Global Temperature")
-p3 = plot(
-    [agent_df[!, aggname(white, total)], agent_df[!, aggname(black, total)]],
-    legend = false,
-    ylabel = "Population",
+model = daisyworld()
+
+# To visualize we need to define the necessary functions for [`plotabm`](@ref).
+# The daisies will obviously be black or white, but the land will have a color
+# that reflects its temperature, with -50 darkest and 100 ᵒC brightest color
+
+daisycolor(a::Daisy) = a.breed
+const landcolor = cgrad(:thermal)
+daisycolor(a::Land) = landcolor[(a.temperature+50)/150]
+
+# And we plot daisies as circles, and land patches as squares
+daisyshape(a::Daisy) = :circle
+daisysize(a::Daisy) = 8
+daisyshape(a::Land) = :square
+daisysize(a::Land) = 13.5
+
+# Notice that we want to ensure that the `Land` patches are always plotted first.
+plotsched = by_type((Land, Daisy), false)
+
+plotkwargs = (
+    ac = daisycolor, am = daisyshape, as = daisysize,
+    scheduler = plotsched, aspect_ratio = 1, size = (600, 600), showaxis = false,
 )
-plot(p1, p2, p3, layout = (3, 1), size = (500, 800))
+
+p = plotabm(model; plotkwargs...)
+
+# And after a couple of steps
+step!(model, agent_step!, model_step!, 5)
+p = plotabm(model; plotkwargs...)
+
+# Let's do some animation now
+Random.seed!(165) # hide
+model = daisyworld()
+anim = @animate for i in 0:30
+    p = plotabm(model; plotkwargs...)
+    title!(p, "step $(i)")
+    step!(model, agent_step!, model_step!)
+end
+gif(anim, "daisyworld.gif", fps = 3)
+
+# Running this animation for longer hints that this world achieves quasi-equilibrium
+# for some input parameters, where one `breed` does not totally dominate the other.
+# Of course we can check this easily through data collection.
+# Notice that here we have to define a function `breed` that gives the daisy's `breed`
+# field. We cannot use just `:breed` to automatically find it, because in this mixed
+# agent model, the `Land` doesn't have any `breed`.
+# %% # src
```

Comment:
Not sure what this command is meant to to, but currently it's being displayed in the documentation text.

Reply:
It isn't, it was misstyped because you have to have no space between `#` and `src`